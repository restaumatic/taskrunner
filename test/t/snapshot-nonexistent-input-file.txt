git init -q

# TODO: the golden contains entire source of hash-files.sh, which is unfortunate, but hard to fix easily
snapshot nonexistent-file.txt
