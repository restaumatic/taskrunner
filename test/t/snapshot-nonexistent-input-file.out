-- output:
[toplevel] stderr | cp: cannot stat '.git/index': No such file or directory
[toplevel] error | snapshot command failed with exception: readCreateProcess: bash "-c" "# Script called from main executable to compute hash of git pathspecs\n\nset -euo pipefail\n\nGIT_INDEX_FILE=\"$(mktemp)\"\ntrap 'rm -f \"${GIT_INDEX_FILE}\"' 0 1 2 3 15\nexport GIT_INDEX_FILE\ncp \"$(git rev-parse --git-dir)/index\" \"${GIT_INDEX_FILE}\"\ntree=\"$(git add \"$@\" && git write-tree)\" || exit 1\ngit ls-tree -r --full-name \"$tree\" \"$@\"\n" "hash-files" "nonexistent-file.txt" (exit 1): failed
-- exit code: 1
